apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/rubricSettingsModal.ejs

```diff
@@ -0,0 +1,276 @@
+                </td>
+                <td style="max-width: 5rem">
+                  <input type="number" class="form-control js-rubric-item-points" step="any" required>
+                </td>
+                <td>
+                  <input type="text" class="form-control js-rubric-item-description" required maxlength="100">
+                </td>
+                <td>
+                  <label class="js-rubric-item-explanation">
+                    <button type="button" class="btn btn-sm js-rubric-item-long-text-field">
+                      <i class="fas fa-pencil"></i>
+                    </button>
+                  </label>
+                </td>
+                <td>
+                  <label class="js-rubric-item-grader-note">
+                    <button type="button" class="btn btn-sm js-rubric-item-long-text-field">
+                      <i class="fas fa-pencil"></i>
+                    </button>
+                  </label>
+                </td>
+                <td>
+                  New
+                </td>
+              </tr>
+            </template>
+            <% if (variant.params || variant.true_answer || submission?.submitted_answer) { %>
+            <div class="small form-text text-muted">
+              Rubric items may use these entries, which are replaced with the corresponding values for the student variant (click to copy):
+              <ul style="max-height: 7rem; overflow-y: auto;">
+                <% [
+                   ...Object.keys(variant.params || {}).map((key) => ['params', key]),
+                   ...Object.keys(variant.true_answer || {}).map((key) => ['correct_answers', key]),
+                   ...Object.keys(submission?.submitted_answer || {}).map((key) => ['submitted_answers', key]),
+                   ].forEach(([group, key]) => { %>
+                <li><code class="js-copy-on-click"
+                          data-clipboard-text="{{<%= group %>.<%= key %>}}"
+                          >{{<%= group %>.<%= key %>}}</code></li>
+                <% }); %>
+              </ul>
+            </div>
+            <% } %>
+          </div>
+          <hr/>
+          <div class="form-check">
+            <label class="form-check-label">
+              <input class="form-check-input" type="checkbox" value="true"
+                     name="tag_for_manual_grading">
+              Require all graded submissions to be manually graded/reviewed
+            </label>
+          </div>
+        </div>
+        <div class="js-settings-error-alert-placeholder"></div>
+        <div class="modal-footer">
+          <% if (authz_data.has_course_instance_permission_edit) { %>
+          <% if (locals.rubric_data) { %>
+          <button type="button" class="btn btn-link btn-sm js-disable-rubric-button mr-auto">
+            Disable rubric
+          </button>
+          <% } %>
+          <button type="submit" class="btn btn-primary">
+            <%= locals.rubric_data ? 'Update rubric' : 'Set rubric' %>
```

Comment:
I think I'd prefer just "Save" in all cases. I was initially confused about "Set rubric".